hoomd/hpmc/ComputeSDF.h

```diff
@@ -154,16 +155,19 @@ template<class Shape> class ComputeSDF : public Compute
     double m_xmax;                                   //!< Maximum lambda value
     double m_dx;                                     //!< Histogram step size
 
-    std::vector<unsigned int> m_hist; //!< Raw histogram data
-    std::vector<double> m_sdf;        //!< Computed SDF
+    std::vector<double> m_hist; //!< Raw histogram data
+    std::vector<double> m_sdf;  //!< Computed SDF
 
     Scalar m_last_max_diam; //!< Last recorded maximum diameter
+    unsigned int m_instance = 0;
 
     //! Zero the histogram counts
     void zeroHistogram();
 
     //! Add to histogram counts
     void countHistogram(uint64_t timestep);
+    void countHistogramBinarySearch(uint64_t timestep);
+    void countHistogramBruteForce(uint64_t timestep);
```

Comment:
```suggestion
    void countHistogramLinearSearch(uint64_t timestep);
```
Let's name this based on a more descriptive title.